tests/Plotly.NET.Tests/HtmlCodegen/ChartLayout.fs

```diff
@@ -0,0 +1,78 @@
+﻿module Tests.ChartLayout
+
+open Expecto
+open Plotly.NET
+open TestUtils
+
+let axisStylingChart =
+    let x = [1.; 2.; 3.; 4.; 5.; 6.; 7.; 8.; 9.; 10.; ]
+    let y = [2.; 1.5; 5.; 1.5; 3.; 2.5; 2.5; 1.5; 3.5; 1.]
+    let plot1 =
+        Chart.Point(x,y)
+        |> Chart.withX_AxisStyle ("X axis title quack quack", MinMax = (-1.,10.))
+        |> Chart.withY_AxisStyle ("Y axis title boo foo", MinMax = (-1.,10.))
+    plot1
+
+
+[<Tests>]
+let ``Axis styling tests`` =
+    testList "Axis styling tests" [
+        testCase "X With axis has title" ( fun c ->
+            "X axis title quack quack"
```

Comment:
I think you can be a little stricter in all of these cases. Something like this: "\"title\":\"X axis title quack quack\""